cmd/nerdctl/stats_linux_test.go

```diff
@@ -32,9 +33,13 @@ func TestStats(t *testing.T) {
 		t.Skip("test skipped for rootless containers on cgroup v1")
 	}
 	testContainerName := testutil.Identifier(t)
+	randomTestContainerName := fmt.Sprintf("%s-random", testContainerName)
```

Comment:
What does "random" mean in this context?

Reply:
Just typo, I will fix it